security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -12,6 +12,34 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-47"]
+== OpenShift Compliance Operator 0.1.47
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.47:
+
+* link:https://access.redhat.com/errata/RHBA-2022:0014[RHBA-2022:0014 OpenShift Compliance Operator Bug Fix and Enhancement Update]
```

Comment:
Not worth holding up the PR, but:

This title might be improved by following the same conventions as the OCP asynchronous release notes. 

For example:

`RHBA-2021:5003 - OpenShift Container Platform 4.9.11 bug fix and security update`
* hypen between advisory number and title
* sentence case
* includes product version number

See https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-12